nat-lab/tests/test_derp_connect.py

```diff
@@ -15,14 +15,40 @@
 DERP3_IP = str(DERP_TERTIARY.ipv4)
 
 
+def _build_parameters(
+    connection_tag: List[ConnectionTag], enable_poll: bool
+) -> List[SetupParameters]:
+    features = default_features()
+    if enable_poll:
+        assert features
+        features.derp = FeatureDerp(
+            tcp_keepalive=120,
+            derp_keepalive=120,
+            poll_keepalive=15,
+            enable_polling=False,
+            use_built_in_root_certificates=False,
```

Comment:
it has `if enable_poll` and `enable_polling=False`. Natlab test in this case doesn't test the new feature which is understandable since derp image is still of v1, however with the separation I've written in another comment, natlab test could test with bth polling enabled and disabled(as it would only result in server not pinging libtelio). I think it's worth pursuing.